Source/WebKit/WebProcess/GPU/media/RemoteRemoteCommandListener.h

```diff
@@ -48,8 +48,8 @@ class RemoteRemoteCommandListener final
     , public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<RemoteRemoteCommandListener> {
     WTF_MAKE_FAST_ALLOCATED;
 public:
-    static Ref<RemoteRemoteCommandListener> create(WebCore::RemoteCommandListenerClient&, WebProcess&);
-    RemoteRemoteCommandListener(WebCore::RemoteCommandListenerClient&, WebProcess&);
+    static Ref<RemoteRemoteCommandListener> create(WebCore::RemoteCommandListenerClient&);
+    RemoteRemoteCommandListener(WebCore::RemoteCommandListenerClient&);
```

Comment:
This time, the opposite:

```suggestion
    explicit RemoteRemoteCommandListener(WebCore::RemoteCommandListenerClient&);
```